frontstage/views/account/account_survey_share.py

```diff
@@ -201,9 +201,15 @@ def share_survey_email_entry(session):
 @jwt_authorization(request)
 def share_survey_post_email_entry(session):
     form = AccountSurveyShareRecipientEmailForm(request.values)
-    if not form.validate():
+    party_id = session.get_party_id()
+    respondent_details = party_controller.get_respondent_party_by_id(party_id)
+    if not form.validate() or respondent_details['emailAddress'].lower() == form.data['email_address'].lower():
+        if respondent_details['emailAddress'].lower() == form.data['email_address'].lower():
+            errors = {'email_address': ['You can not share surveys to yourself.']}
```

Comment:
```suggestion
            errors = {'email_address': ['You can not share surveys with yourself.']}
```

Very small suggestion but generally I'd consider it "share with" and "transfer to". Jeremy might have opinions on this though!

Reply:
👍